File: bucles.hanssy.js
// Definir una clase Persona
class Persona {
    // Atributos de la clase
    constructor(nombre, edad) {
        this.nombre = nombre;  // Asignar el nombre al atributo
        this.edad = edad;      // Asignar la edad al atributo
    }
}

// Crear instancias de la clase Persona
const a = new Persona("Esteban", 28);
const b = new Persona("Hans", 27);
const c = new Persona("Jhon", 41);
const d = new Persona("Juan Pablo", 23);

// Imprimir los nombres y edades
console.log(a.nombre + " " + a.edad);
console.log(b.nombre + " " + b.edad);
console.log(c.nombre + " " + c.edad);
console.log(d.nombre + " " + d.edad);

File: bucles.hanssy.py
# Lista de nombres
nombres = ['Esteban', 'Hans', 'Jhon', 'Juan Pablo']

# Imprimir los nombres
for nombre in nombres:
    print(nombre)

# Diccionarios (en Python usamos diccionarios)
personas = []  # Lista vacía

# Crear diccionarios (similares a objetos en JavaScript)
a = {'nombre': 'Esteban', 'Edad': 28}
b = {'nombre': 'Hans', 'Edad': 27}
c = {'nombre': 'Jhon', 'Edad': 41}
d = {'nombre': 'Juan Pablo', 'Edad': 23}

# Agregar los diccionarios a la lista
personas.append(a)
personas.append(b)
personas.append(c)
personas.append(d)

# Imprimir los nombres y edades
for persona in personas:
    print(persona['nombre'], persona['Edad'])

# Operar con valores de los diccionarios
suma_edades = 0
for persona in personas:
    suma_edades += persona['Edad']  # Sumar las edades

print("La suma total de las edades es:", suma_edades)